python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,521 @@
+# =============================================================================
+# The END
+# =============================================================================
+"""
+
+    ## Returns file names which should be copyed to shared area
+    def inputfiles(self):
+        "Returns file names which should be copyed to shared area"
+        return [ LocalFile(f.name) for f in self.scripts ] + [ LocalFile(f.name) for f in self.imports ]
+    
+    ## Returns  wrapper file which run at WN
+    def getWrapper(self, data_file):
+        "Returns  wrapper file which run at WN"
+
+        the_script    = self.layout.format (
+            scripts   = [ os.path.join ( f.subdir , os.path.basename ( f.name ) ) for f in self.scripts ] , 
+            arguments = self.arguments  ,
+            imports   = [ os.path.join ( f.subdir , os.path.basename ( f.name ) ) for f in self.imports ] ,
+            datafile  = data_file ,
+            command   = self.commands    
+            )
+        return the_script
+
+
+# =============================================================================
+## @class ostap
+#  Helper class to define the main properties of Ostap application
+#  - the scripts to be executed 
+#  - bender commands to be executed
+#  - other arguments
+#
+#  The scripts are executed within ``ostap'' context
+#  Command line arguments ``--no-color'' and ``--batch'' are added automatically
+#
+#  The actual command to be executed is:
+#  @code 
+#  > ostap [ scripts [scripts [scripts ...  [ arguments ] --no-color --batch   [ --command  [ commands [ commands [ commands
+#  @endcode
+#  Usage:
+#  @code
+#  j.application = BenderBox ( tool = Ostap( scripts   = ['path_to_script/the_script.py']  ,
+#                                            arguments = [ '--no-canvas' ]  ,
+#                                            commands  = [ 'print dir()' ]  ) ) 
+#  @encode
+#  @author Vladimir ROMANOVSKY Vladimir.Romanovskiy@cern.ch
+#  @author Vanya BELYAEV  Ivan.Belyaev@itep.ru       
+class ostap(GangaObject):
+    """Helper class to define the main properties of Ostap application
+    - the scripts to be executed 
+    - ostap commands to be executed
+    - other arguments
+    
+    The scripts are executed within ``ostap'' context
+    Command line arguments ``--no-color'' and ``--batch'' are added automatically
+
+    The actual command to be executed is:
+    > ostap [ scripts [scripts [scripts ...  [ arguments ] --no-color --batch   [ --command  [ commands [ commands [ commands
+
+    ======
+    Usage:
+    ======
+    j.application = BenderBox ( tool = ostap( scripts   = ['path_to_script/the_script.py']  ,
+                                              arguments = [ '--no-canvas' ]  ,
+                                              commands  = [ 'print dir()' ]  ) )                                                 
+    """
+    _schema = Schema(Version(1, 0), {
+        'scripts'   : FileItem   (
```

Comment:
@egede  Thank you!   It supposed to be a plain local file/script/code.  Might be due to a lack of imagination, but I can't see what one can gain by converting  python code into  DiracFile.  Properly-written code for Bender&bender&ostap  is usually very compact  [it is a main idea fro all these cases: very compact and well readable code for analysis].